descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,124 @@
+{
+  "id": "${artifactId}-${version}",
+  "name": "Module KB EBSCO Module",
```

Comment:
Can we rename this to "EBSCO KB Broker" or probably an even better name? Thanks!

Reply:
yes, sure. will rename it